dbms/tests/integration/helpers/client.py

```diff
@@ -89,3 +93,20 @@ def get_answer(self):
             raise QueryRuntimeException('Client failed! Return code: {}, stderr: {}'.format(self.process.returncode, stderr))
 
         return stdout
+
+
+    def get_error(self):
+        self.process.wait()
+        self.stdout_file.seek(0)
+        self.stderr_file.seek(0)
+
+        stdout = self.stdout_file.read()
+        stderr = self.stderr_file.read()
+
+        if self.timer is not None and not self.process_finished_before_timeout and not self.ignore_error:
+            raise QueryTimeoutExceedException('Client timed out!')
+
+        if (self.process.returncode == 0):
+            raise QueryRuntimeException('Client expected to be failed but was succeeded! stdout: {}'.format(stdout))
```

Comment:
"was" is not grammatically correct and should be omitted.  Would you please correct to the following:
"Client expected to be failed but succeeded"

Reply:
Sure. https://github.com/yandex/ClickHouse/pull/5573